src/sql/azdata.proposed.d.ts

```diff
@@ -1486,6 +1486,17 @@ declare module 'azdata' {
 			 */
 			viewModel: DesignerViewModel;
 		}
+
+		export interface GeneratePreviewReportResult {
```

Comment:
I'd suggest just naming this `PreviewReport` - the extension api typically doesn't have the concept of requests and responses.